Feat(eos_designs): Automatic uplink/downlink/mlag peer-link allocation (#1758)
aristanetworks/ansible-avd: ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

    # IPv4 subnet to use to connect to uplink switches.
    uplink_ipv4_pool: < IPv4_address/Mask  >

    # Local uplink interfaces (list). | Optional

>     # If uplink_interfaces is not defined, platform-specific defaults will be used instead.

Author: ```suggestion
    # If uplink_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.
```

>     # If uplink_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.

Reviewer: The input variables are not "facts".
```suggestion
    # If uplink_interfaces is not defined, platform-specific defaults (defined under default_interfaces) will be used instead.
```

>     # Please note that these defaults are not defined by default - you should define these yourself.

Reviewer: ```suggestion
    # Please note that default_interfaces are not defined by default - you should define these yourself.
```

Author: Done.

  node_groups:
    < node-group-name >:
      nodes:
        # Uplink switches interfaces (list), interface located on uplink switch. | Optional

>         # If uplink_switch_interfaces is not defined, platform-specific defaults will be used instead.

Author: ```suggestion
        # If uplink_switch_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.
```

>         # If uplink_switch_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.
>         # Please note that these defaults are not defined in default_platform_settings - you should define these yourself.

Reviewer: ```suggestion
        # If uplink_switch_interfaces is not defined, platform-specific defaults (defined under default_interfaces) will be used instead.
        # Please note that default_interfaces are not defined by default - you should define these yourself.
```

Author: Done.

    # loopbacks from spine perspective and avoid suboptimal routing via peerlink for control plane traffic.
    mlag_ibgp_origin_incomplete: < true | false | default -> true >

    # MLAG interfaces (list) | Optional, even when MLAG leafs present in topology.

>     # If mlag_interfaces is not defined, platform-specific defaults will be used instead.

Author: ```suggestion
    # If mlag_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.
```

>     # If mlag_interfaces is not defined, platform-specific defaults (defined by default_interfaces fact) will be used instead.

Reviewer: ```suggestion
    # If mlag_interfaces is not defined, platform-specific defaults (defined under default_interfaces) will be used instead.
```

Author: Done.

>     # Please note that these defaults are not defined in default_platform_settings - you should define these yourself.

Reviewer: ```suggestion
    # Please note that these default_interfaces are not defined by default - you should define these yourself.
```
